frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -93,7 +93,7 @@ const PodStatus: React.FC<PodStatusProps> = ({
   const chartDonut = React.useMemo(() => {
     return (
       <ChartDonut
-        ariaTitle={`${title} ${subTitle}`}
+        ariaTitle={`${title} ${subTitle || ''}`}
```

Comment:
will it be better to pass the required subTitle from the component https://github.com/openshift/console/blob/master/frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewListItem.tsx#L52 ?

and it'll be nice to show `0 Pod` over `0`, WDYT

Reply:
will pass empty subtitle thank you

regarding `0 Pod` doing that will result in the subtitle shown in the center as well so avoiding it 
![image](https://user-images.githubusercontent.com/20089340/147556746-6f4713c0-efb5-464d-b74c-514d6de42370.png)

Reply:
let's do, passing empty subtitle is not good  https://github.com/openshift/console/pull/10715#discussion_r775848436